scss/functions/_get-color.scss

```diff
@@ -5,23 +5,14 @@
   // add the logic here
   // use if else to check if the type and level are valid
   // if they are valid, return the color
+  @if map.has-key($colors, $type, $level) {
+    $color: map.get(map.get($colors, $type), $level);
+    @return $color;
```

Comment:
masih kurang tepat ya kak. kamu paling tidak harus melakukan 2 pengkondisian untuk pengecekan `$type` dan `$level`
sekedar clue, kamu bisa memakai `@if map.has-key(....) and @if map.has-key(...){`

Reply:
mas, bagaimana cara saya commitnya agar langsung dibawah feedbacknya mas, biar commitnya gak bercabang?